trusona-sdk/src/main/java/com/trusona/sdk/Trusona.java

```diff
@@ -68,25 +69,52 @@
 
   //TODO: Remove these intermediary constructors after everything is delegated to clients.
   Trusona(ServiceGenerator sg, ApiCredentials apiCredentials) {
-    this(sg, apiCredentials, new TrusonaficationClient(sg), new DevicesClient(sg), new UsersClient(sg));
+    this(sg, apiCredentials, new TrusonaficationClient(sg), new DevicesClient(sg),
+        new UsersClient(sg));
   }
 
   Trusona(TrusonaEnvironment environment, ApiCredentials apiCredentials) {
     this(ServiceGenerator.create(
-      EnvironmentFactory.getEnvironment(environment),
-      apiCredentials,
-      new DefaultHmacSignatureGenerator()),
-         apiCredentials);
+        EnvironmentFactory.getEnvironment(environment),
+        apiCredentials,
+        new DefaultHmacSignatureGenerator()),
+        apiCredentials);
   }
 
+  Trusona(CustomEnvironment environment, ApiCredentials apiCredentials) {
+    this(ServiceGenerator.create(
+        environment,
+        apiCredentials,
+        new DefaultHmacSignatureGenerator()),
+        apiCredentials);
+  }
+
+  /**
+   * Create a Trusona object using an endpoint URL
+   *
+   * @param token       Your API token, provided by Trusona
+   * @param secret      Your API secret, provided by Trusona
+   * @param endpointUrl The Trusona endpoint URL you would like to point at
+   */
+  public Trusona(String token, String secret, String endpointUrl) {
+    this(EnvironmentFactory.getCustomEnvironment(endpointUrl), new ApiCredentials(token, secret));
+  }
```

Comment:
Once I went back to the Gateway, I realized I needed a way to actually make a `Trusona` object with the `endpointUrl` value so I came back and added this.